### ISSUE TITLE
404 page error with jekyll-seo-tag 2.6.1

### ISSUE THREAD

**Issue by DotIN13** [samarsault/plainwhite-jekyll#64]
I am currently getting the following error with jekyll 4.1.0 and jekyll-seo-tag 2.6.1.

```log
Configuration file: /home/dotin13/Documents/GitHub/plainwhite-jekyll/_config.yml
 Theme Config file: /home/dotin13/Documents/GitHub/plainwhite-jekyll/_config.yml
            Source: /home/dotin13/Documents/GitHub/plainwhite-jekyll
       Destination: /home/dotin13/Documents/GitHub/plainwhite-jekyll/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
  Liquid Exception: undefined method `[]' for nil:NilClass in 404.html/#excerpt
  Liquid Exception: undefined method `[]' for nil:NilClass in 404.html
                    ------------------------------------------------
      Jekyll 4.1.0   Please append `--trace` to the `serve` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
Traceback (most recent call last):
	142: from /home/dotin13/gems/bin/jekyll:23:in `<main>'
	141: from /home/dotin13/gems/bin/jekyll:23:in `load'
	140: from /home/dotin13/gems/gems/jekyll-4.1.0/exe/jekyll:15:in `<top (required)>'
	139: from /home/dotin13/gems/gems/mercenary-0.4.0/lib/mercenary.rb:21:in `program'
	138: from /home/dotin13/gems/gems/mercenary-0.4.0/lib/mercenary/program.rb:44:in `go'
	137: from /home/dotin13/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `execute'
	136: from /home/dotin13/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `each'
	135: from /home/dotin13/gems/gems/mercenary-0.4.0/lib/mercenary/command.rb:221:in `block in execute'
	134: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/commands/serve.rb:86:in `block (2 levels) in init_with_program'
	133: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/command.rb:91:in `process_with_graceful_fail'
	132: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/command.rb:91:in `each'
	131: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/command.rb:91:in `block in process_with_graceful_fail'
	130: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/commands/build.rb:36:in `process'
	129: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/commands/build.rb:65:in `build'
	128: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/command.rb:28:in `process_site'
	127: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/site.rb:80:in `process'
	126: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/site.rb:208:in `render'
	125: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/site.rb:523:in `render_pages'
	124: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/site.rb:523:in `each'
	123: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/site.rb:524:in `block in render_pages'
	122: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/site.rb:532:in `render_regenerated'
	121: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/renderer.rb:63:in `run'
	120: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/renderer.rb:80:in `render_document'
	119: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/renderer.rb:127:in `render_liquid'
	118: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:36:in `render!'
	117: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time'
	116: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!'
	115: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes'
	114: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!'
	113: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts'
	112: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!'
	111: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!'
	110: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render'
	109: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling'
	108: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render'
	107: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render'
	106: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output'
	105: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/tags/include.rb:210:in `render'
	104: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/context.rb:123:in `stack'
	103: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/tags/include.rb:212:in `block in render'
	102: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/inclusion.rb:16:in `render'
	101: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:36:in `render!'
	100: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time'
	 99: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!'
	 98: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes'
	 97: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!'
	 96: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts'
	 95: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!'
	 94: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!'
	 93: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render'
	 92: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling'
	 91: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render'
	 90: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render'
	 89: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output'
	 88: from /home/dotin13/gems/gems/jekyll-seo-tag-2.6.1/lib/jekyll-seo-tag.rb:36:in `render'
	 87: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!'
	 86: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render'
	 85: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling'
	 84: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render'
	 83: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:82:in `render'
	 82: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output'
	 81: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/tags/if.rb:43:in `render'
	 80: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/context.rb:123:in `stack'
	 79: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/tags/if.rb:44:in `block in render'
	 78: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/tags/if.rb:44:in `each'
	 77: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/tags/if.rb:45:in `block (2 levels) in render'
	 76: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/condition.rb:46:in `evaluate'
	 75: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/condition.rb:46:in `loop'
	 74: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/condition.rb:47:in `block in evaluate'
	 73: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/condition.rb:114:in `interpret_condition'
	 72: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/context.rb:159:in `evaluate'
	 71: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/variable_lookup.rb:38:in `evaluate'
	 70: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/variable_lookup.rb:38:in `each_index'
	 69: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/variable_lookup.rb:48:in `block in evaluate'
	 68: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/context.rb:197:in `lookup_and_evaluate'
	 67: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/drops/drop.rb:47:in `[]'
	 66: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/drops/drop.rb:47:in `public_send'
	 65: from /home/dotin13/gems/gems/jekyll-seo-tag-2.6.1/lib/jekyll-seo-tag/drop.rb:81:in `description'
	 64: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/drops/drop.rb:47:in `[]'
	 63: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/drops/drop.rb:47:in `public_send'
	 62: from /usr/lib/ruby/2.7.0/forwardable.rb:235:in `excerpt'
	 61: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/page.rb:226:in `excerpt'
	 60: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/page.rb:226:in `new'
	 59: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/page_excerpt.rb:13:in `initialize'
	 58: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/renderer.rb:63:in `run'
	 57: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/renderer.rb:80:in `render_document'
	 56: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/renderer.rb:127:in `render_liquid'
	 55: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:36:in `render!'
	 54: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time'
	 53: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!'
	 52: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes'
	 51: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!'
	 50: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts'
	 49: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!'
	 48: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!'
	 47: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render'
	 46: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling'
	 45: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render'
	 44: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render'
	 43: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output'
	 42: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/tags/include.rb:210:in `render'
	 41: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/context.rb:123:in `stack'
	 40: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/tags/include.rb:212:in `block in render'
	 39: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/inclusion.rb:16:in `render'
	 38: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:36:in `render!'
	 37: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:70:in `measure_time'
	 36: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:37:in `block in render!'
	 35: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:63:in `measure_bytes'
	 34: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:38:in `block (2 levels) in render!'
	 33: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:59:in `measure_counts'
	 32: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/liquid_renderer/file.rb:39:in `block (3 levels) in render!'
	 31: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!'
	 30: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render'
	 29: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling'
	 28: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render'
	 27: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render'
	 26: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output'
	 25: from /home/dotin13/gems/gems/jekyll-seo-tag-2.6.1/lib/jekyll-seo-tag.rb:36:in `render'
	 24: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!'
	 23: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render'
	 22: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling'
	 21: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render'
	 20: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:82:in `render'
	 19: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output'
	 18: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/tags/if.rb:43:in `render'
	 17: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/context.rb:123:in `stack'
	 16: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/tags/if.rb:44:in `block in render'
	 15: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/tags/if.rb:44:in `each'
	 14: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/tags/if.rb:45:in `block (2 levels) in render'
	 13: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/condition.rb:46:in `evaluate'
	 12: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/condition.rb:46:in `loop'
	 11: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/condition.rb:47:in `block in evaluate'
	 10: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/condition.rb:114:in `interpret_condition'
	  9: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/context.rb:159:in `evaluate'
	  8: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/variable_lookup.rb:38:in `evaluate'
	  7: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/variable_lookup.rb:38:in `each_index'
	  6: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/variable_lookup.rb:48:in `block in evaluate'
	  5: from /home/dotin13/.gem/ruby/2.7.0/gems/liquid-4.0.3/lib/liquid/context.rb:197:in `lookup_and_evaluate'
	  4: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/drops/drop.rb:47:in `[]'
	  3: from /home/dotin13/gems/gems/jekyll-4.1.0/lib/jekyll/drops/drop.rb:47:in `public_send'
	  2: from /home/dotin13/gems/gems/jekyll-seo-tag-2.6.1/lib/jekyll-seo-tag/drop.rb:27:in `title?'
	  1: from /home/dotin13/gems/gems/jekyll-seo-tag-2.6.1/lib/jekyll-seo-tag/drop.rb:50:in `title'
/home/dotin13/gems/gems/jekyll-seo-tag-2.6.1/lib/jekyll-seo-tag/drop.rb:43:in `page_title': undefined method `[]' for nil:NilClass (NoMethodError)
```

Adding `excerpt: '404'` would fix this issue. But still not sure its cause.

Any thoughts appreciated.

**Comment by DotIN13** [comment]
Looks like an issue related to jekyll. Ref: [jekyll-seo-tag #401](https://github.com/jekyll/jekyll-seo-tag/issues/401)

**Comment by DotIN13** [comment]
The issue was fixed in jekyll 4.1.1. Closing.
